stdlib/printexc.mli

```diff
@@ -193,7 +193,7 @@ val backtrace_slots : raw_backtrace -> backtrace_slot array option
     recent function call, raise, or primitive [get_backtrace] call in
     the trace.
 
-    Some possible reasons for returning [None] are as follow:
+    Some possible reasons for returning [None] are as follows:
```

Comment:
Some style guides ([e.g.](http://www.bristol.ac.uk/arts/exercises/grammar/grammar_tutorial/page_04.htm)) recommend that the clause before a colon should be a complete sentence.  I think that's a good rule to follow.